seed/seed20.go

```diff
@@ -762,3 +763,34 @@ func (s *seed20) Iter(f func(sn *Snap) error) error {
 	}
 	return nil
 }
+
+func (s *seed20) LoadAutoImportAssertions(commitTo func(*asserts.Batch) error) {
+	if s.model.Grade() == asserts.ModelDangerous {
```

Comment:
I don't know if this was pointed out, but could we do a reverse-logic here?
```
if s.model.Grade() != asserts.ModelDangerous {
    return
}
```

Reply:
that seems a bit cleaner indeed

Reply:
Agree.
But before changing it. @mvo5 raised question if this should be limited to the `dangerous` model only.
Part of me agrees with Michael. Motivation to limit it was to raise less controversy for initial PR, and potentially extend it to other models later.
@pedronis what is your opinion to allow this for all the models?
We can still limit system-user creation to dangerous models only.